x-pack/plugins/maps/public/connected_components/right_side_controls/layer_control/layer_toc/toc_entry/toc_entry_actions_popover/toc_entry_actions_popover.tsx

```diff
@@ -210,6 +213,10 @@ export class TOCEntryActionsPopover extends Component<Props, State> {
             } else {
               this.props.enablePointEditing(this.props.layer.getId());
             }
+
+            this.props.updateSourceProp(this.props.layer.getId(), 'applyGlobalQuery', false);
```

Comment:
Please remove calls to set `applyGlobalQuery`, `applyGlobalTime`, and `query`. Opening "edit features" should not change any of these values. Instead, these values should be ignored while in "edit features" mode but keep there values so when exiting "edit features", the map is in the same state as before.